apps/jobbrowser/src/jobbrowser/templates/job_browser.mako

```diff
@@ -3048,6 +3091,22 @@ ${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
           });
         }
       }
+
+      self.onTableRowClick = function (event, id) {
+        var openInNewTab = event && (event.which === 2 || event.metaKey || event.ctrlKey);
```

Comment:
nit: 
Could make it a bit easier to understand with `var middleMouseButton = 2`

Reply:
Good one, I'll add it.